packages/react-button/src/components/MenuButton/useMenuButtonClasses.ts

```diff
@@ -16,7 +17,7 @@ export const useMenuButtonClasses = makeVariantClasses<MenuButtonState, MenuButt
     root: [
       GlobalClassNames.root,
       {
-        // This seems like a bad selector.
+        // This seems likea  bad selector.
```

Comment:
```suggestion
        // This seems like a  bad selector.
```

Reply:
Or remove the selector. It seems bad. :)

Reply:
Thanks! I'll change this.

Reply:
Changed!